recipe/meta.yaml

```diff
@@ -20,14 +20,15 @@ requirements:
     - {{ stdlib("c") }}
     - {{ compiler('cxx') }}
     - cmake >=2.8.12
+    - sysroot_linux-64 2.28
```

Comment:
This should no longer be used in recipes, the `stdlib` function is the new way to include/set it.
```suggestion
```